.github/workflows/pr.yaml

```diff
@@ -86,8 +86,6 @@ jobs:
       - uses: taiki-e/install-action@v2
         with:
           tool: nextest@0.9.98
-      - name: Install gdb
```

Comment:
I think we need this so we get stack dumps when a test times out (see the scripts dir)

Reply:
I made a change to install it inside the script, that way it's not tied to CI, and it's probably more useful. It will also only be installed if it's about to be used.